src/nitclk/templates/session.py.mako

```diff
@@ -49,15 +49,15 @@ helper.add_attribute_rep_cap_tip(attributes[attribute], config)
 %   endif
 % endfor
 
-    def __init__(self, ${config['session_handle_parameter_name']}, encoding='windows-1251'):
+    def __init__(self, session_handle, encoding='windows-1251'):
```

Comment:
Different APIs use different names for the concept of a session handle.

This change makes everything use the same. NI-ModInst uses a different name. NI-TClk uses a different concept altogether. I think this need to be reverted to the way it was (it's also a change in public API).

Reply:
Updated NI-ModInst's and NI-TClk's session.py to mostly revert this rename.